Formula/g/gnu-cobol.rb

```diff
@@ -20,40 +24,97 @@ class GnuCobol < Formula
     sha256 x86_64_linux:   "4bf957f586d155f1034ac90ac24edff1253e9de935336680689d85f0bbdcf45c"
   end
 
+  head do
+    # there's no offical git mirror, so we use the current upstream SVN repo
+    url "https://svn.code.sf.net/p/gnucobol/code/trunk"
+
+    # tools for bootstrapping
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+    depends_on "bison" => :build
+    depends_on "flex" => :build
+    depends_on "gettext" => :build
+    depends_on "help2man" => :build
+    depends_on "libtool" => :build
+    depends_on "texinfo" => :build
+
+    # head features multiple indexed handlers
```

Comment:
because those are for features that the stable version does not have at all, I'll adjust the comment to make that more clear

Reply:
And will they be available in a future stable release? If so, please also indicate when they will be added so that the dependencies can be adjusted then.